test/performance/odp_packet_gen.c

```diff
@@ -1798,7 +1798,7 @@ static int tx_thread(void *arg)
 	/* Start TX burst at different per thread offset */
 	t1_nsec = odp_time_to_ns(t1) + gap_nsec + (tx_thr * gap_nsec / num_tx);
 
-	while (ret == 0) {
+	while (true) {
```

Comment:
This change shouldn't be necessary anymore after PR #2072. `ret != 0` only in case `odp_pktout_send()` returned failure.

Reply:
Ack